cli.go

```diff
@@ -375,3 +391,24 @@ func printResult(out io.Writer, result *Result, mode DisplayMode, withStats bool
 		}
 	}
 }
+
+// buildDdlStatements build batched statement only if all statements are DDL statements
+func buildDdlStatements(input string) Statement {
+	var ddls []string
+	for _, separated := range separateInput(input) {
+		stmt, err := BuildStatement(separated.statement)
+		if err != nil {
+			return nil
+		}
+
+		switch ddl := stmt.(type) {
+		case *DdlStatements:
```

Comment:
Do we need to check `DdlStatements` here although `BuildStatement()` doesn't populate such statement?

Reply:
It is not needed.
Fixed.